nmosconnection/api.py

```diff
@@ -33,7 +33,7 @@
 CONN_APINAME = "connection"
 CONN_APIVERSIONS = ["v1.0", "v1.1"]
 
-DEVICE_ROOT = '/' + CONN_APINAMESPACE + '/' + CONN_APINAME + '/'
+CONN_ROOT = '/' + CONN_APINAMESPACE + '/' + CONN_APINAME + '/'
```

Comment:
Annoyingly this is imported here: https://github.com/bbc/rd-ips-cm-ipstudio-driver/blob/master/nmosconnectiondriver/httpIpstudioDriver.py#L7. We'll need to handle that before merging.